x-pack/filebeat/input/cel/input.go

```diff
@@ -831,6 +833,16 @@ func checkRedirect(cfg *ResourceConfig, log *logp.Logger) func(*http.Request, []
 	}
 }
 
+// retryErrorHandler returns a retryablehttp.ErrorHandler that will log retry resignation
+// but return the last retry attempt's response and a nil error so that the CEL code
+// can evaluate the response status itself.
+func retryErrorHandler(max int, log *logp.Logger) retryablehttp.ErrorHandler {
+	return func(resp *http.Response, err error, numTries int) (*http.Response, error) {
```

Comment:
Do we want to swallow this `err` coming from [Client.Do(req)](https://github.com/hashicorp/go-retryablehttp/blob/main/client.go#L740-L747) without logging?

Reply:
Good catch, we should return this error.